### PR TITLE
update helm repo and chart name

### PR DESCRIPTION
**Pull request by cpanato**
- We released a new helm chart repo and changed the chart name this PR updates those

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]

File: kubernetes/cluster_deploy.go [hunk 1]
```diff
@@ -45,26 +45,26 @@ func (c *Cluster) Deploy(options *ltops.DeployOptions) error {
...
 		// Delete the app pods to make sure they reset caches and restart to pick up config changes
-		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,app=mattermost-helm", c.Release()))
+		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,app=mattermost-enterprise-edition", c.Release()))
 		if out, err := cmd.CombinedOutput(); err != nil {
 			return errors.Wrap(err, "unable to restart app server pods, error from kubectl: "+string(out))
 		}
 
 		// Delete the loadtest pods to make sure they pick up config changes
-		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,app=mattermost-helm-loadtest", c.Release()))
+		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,appmattermost-enterprise-edition-loadtest", c.Release()))
```
Comment (lieut-data) [inline kubernetes/cluster_deploy.go]: Is this right? `appmattermost-enterprise-edition-loadtest` vs `app=mattermost-enterprise-edition-loadtest`

**Review by lieut-data — CHANGES_REQUESTED** [review]
Actually, one anomaly below.

File: kubernetes/cluster_deploy.go (hunk 1 above)
Comment (cpanato) [reply kubernetes/cluster_deploy.go]: i will check and rethink :)

**Review by jwilander — APPROVED** [review]
LGTM other than Jesse's comment

File: kubernetes/cluster_deploy.go (hunk 1 above)
Comment (cpanato) [reply kubernetes/cluster_deploy.go]: fixed @lieut-data

**Review by lieut-data — APPROVED** [review]

**Pull request merged**